### PR TITLE
[ISSUE#2810].modify the judge logic when send beat in raftcore

### PR DESCRIPTION
**Pull request by horizonzy**
## What is the purpose of the change

fix-#2810

### CODE REVIEW

File: naming/src/main/java/com/alibaba/nacos/naming/consistency/persistent/raft/RaftCore.java [hunk 1]
```diff
@@ -483,7 +483,10 @@ public void run() {
 
         public void sendBeat() throws IOException, InterruptedException {
             RaftPeer local = peers.local();
-            if (local.state != RaftPeer.State.LEADER && !ApplicationUtils.getStandaloneMode()) {
+            if (ApplicationUtils.getStandaloneMode()){
```
[the remark this answers is not in the document]
Comment (horizonzy) [reply naming/src/main/java/com/alibaba/nacos/naming/consistency/persistent/raft/RaftCore.java:486]: same logic, change || will be better

**Comment by horizonzy** [comment]
> The old logic seems to the beat only Follower member do not send. Whether standalone mode need send beat need to discuss.

send beat in standalone model maybe is not nessesary.

**Comment by KomachiSion** [comment]
> > The old logic seems to the beat only Follower member do not send. Whether standalone mode need send beat need to discuss.
> 
> send beat in standalone model maybe is not nessesary.

Let's change it to `||` logic please and fix the CI.

**Comment by horizonzy** [comment]
see #3047

**Pull request closed without merging**